src/components/Main.js

```diff
@@ -120,6 +120,10 @@ class Main extends React.Component {
     }
   }
 
+  showDefaltMainMenuActions() {
```

Comment:
asi zbytocna funkcia. staci dat do `render()` `const showDefaltMainMenuActions = this.props.tool !== 'route-planner' && this.props.tool !== 'search';` a pouzitat tuto premennu namiesto volania funkcie.